packages/cli/src/commands/capabilities.ts

```diff
@@ -164,6 +164,15 @@ export async function getStandard(client: SmartThingsClient): Promise<Capability
 	return caps.map((capability: CapabilitySummary) => { return { ...capability, namespace: 'st' } })
 }
 
+export async function getAllFiltered(client: SmartThingsClient, filter: string): Promise<CapabilitySummaryWithNamespace[]> {
+	const list = (await Promise.all([getStandard(client), getCustomByNamespace(client)])).flat()
+	if (filter) {
+		filter = filter.toLowerCase()
+		return list.filter(it => it.id.toLowerCase().includes(filter) && it.status !== 'deprecated')
```

Comment:
I would opt for naming the filter param something like `capability` instead of `it`.

Reply:
Changed